physionet-django/templates/navbar.html

```diff
@@ -1,22 +1,195 @@
 {% load static %}
-<nav class="navbar navbar-expand-md navbar-dark bg-dark d-flex justify-content-between" id="mainNav">
-	<label for="navicon" class="navbar-icon">&#9776;</label>
-	<input type="checkbox" id="navicon" class="navbar-check">
 
-	<a id="nav_home" class="navbar-brand" href="{% url 'home' %}">
-		{{ SITE_NAME }}
-	</a>
+<div class="modern-ui nav-wrapper">
+  <nav class="nav">
+    <div class="container my-0 h-100 d-flex justify-content-between align-items-center">
+      <button type="button" class="nav__hamburger circle-button mr-2">
+        <i class="nav__hamburger__list bi bi-list"></i>
+        <i class="nav__hamburger__x bi bi-x"></i>
+      </button>
+      <a href="/"><img src="{% static 'images/logo.png' %}" width="200px" alt="PhysioNet Logo" /></a>
+      <div class="nav__links align-items-center">
+        <a href="/content"><button type="button" class="nav-link">Discover Data</button></a>
+        <a href="/about/publish"><button type="button" class="nav-link">Share Data</button></a>
```

Comment:
These menu items ("about", "share data" etc) should be pulled in from the static pages model, not hardcoded. One the static page management page (e.g. https://physionet.org/console/static-page/2/edit/) there is a item called "nav bar" that determines whether the page appears in the nav bar.